### PR TITLE
Update for MC 1.1

### PR DESCRIPTION
**Pull request by fernferret**
Hey guys, not sure if you were interested in maintaining this repo still, but I found it very useful! I will be continually maintaining it for future versions. Feel free to pull or if not, thanks for the awesome framework!

### CODE REVIEW

**Comment by sk89q** [comment]
Nice work.
- Some of the file comments seem to have been affected by a replace-in-file
- Could you use Oracle syntax conventions (a space after 'if' I believe)?
- Did you have trouble with Maven?
- Would you like commit access?

File: src/main/java/com/sk89q/minerhat/MinecraftClientConnection.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU General Public License
```
Comment (fernferret) [inline src/main/java/com/sk89q/minerhat/MinecraftClientConnection.java:15]: Ha, yea this is kinda funny. can't believe this happened...

**Comment by fernferret** [comment]
> Some of the file comments seem to have been affected by a replace-in-file

Heh, whoops, I'll get those fixed tonight.

> Could you use Oracle syntax conventions (a space after 'if' I believe)?

Oh totally. @main-- would yell at me for not using checkstyle. I'll fix said formatting issues (especially the ifs)

> Did you have trouble with Maven?

I added some default goals (`clean package`), then ran it (`mvn`), but couldn't get it to fire off (`java minerhat.jar`). Am I missing some parameters? I wasn't that concerned, so I just ran it in IntelliJ. I decided I'd get all the new packets in (and it running again, the login packet was a bit out of date :P) before I'd go the maven route, but I'm a big maven fan, so that's definitely on my todo list.

> Would you like commit access?

Well I'd definitely like to keep maintaining this project you've done an awesome job with, but I'm not bothered either way (committing to your repo or my fork).

EDIT: Fixed formatting

**Comment by fernferret** [comment]
A few other features I wanted to add were:
- Ability to log to a file (I've done this locally, just wanted some better config options, currently just dumps to mc.log in the default location)
- Ability to dynamically filter packets displayed to the user (so if you're looking for a specific packet, you'll only see those)

**Comment by sk89q** [comment]
On the Maven issue, don't you have to use `java -jar minerhat.jar` to run it?

**Comment by fernferret** [comment]
I'm not sure I want to talk about that now... (embarrassed) I'd imagine that'd be why... Sleep should have come much earlier than It did last night.

EDIT: It's been so long since I've even run the Bukkit jars, my simple `update.sh` and `run.sh` scripts have crippled my java life.

**Comment by fernferret** [comment]
Oof this PR has been open since 2012, I'm going to close it so it goes out of my dashboard.

**Pull request closed without merging**